pipeline-scripts/release.groovy

```diff
@@ -348,7 +362,7 @@ def void sendReleaseCompleteMessage(Map release, int advisoryNumber, String advi
             "type": "ocp-release",
             "name": "ocp-release",
             "version": releaseName,
-            "nvr": "ocp-release-${releaseName}",
+            "nvr": "ocp-release-${destReleaseTag(releaseName, arch)}",
```

Comment:
If we are including the arch suffix in the nvr field, it should be also included in the release field I think.

Reply:
all I'm doing here is replacing `releaseName` by a `destReleaseTag`, the fact that tag contains arch is just a detail ...

Reply:
Well, what I mean initially is that the value of nvr should match other fields. Because you changed the value of nvr, you also need to update the version-release part.

Unfortunately it looks like we don't have a field for the arch name. Nowadays with multiarch we are sending arch-specific messages. I don't know the effect of appending arch suffix to the nvr but it will look better if we include the arch in the version field as well or add a new field arch.

Reply:
Assuming it's OK to just add a field for architecture, I'd surely rather do that. I wasn't sure if the consumer of this message would rather differentiate based on the NVR or a separate field. Indeed they might get confused if we release the same NVR twice. Could you find out?

I think I'll just add the `architecture` field for now unless someone says otherwise.